### DISCUSSION TITLE
little enchancement for the webpage

### DISCUSSION THREAD

**Discussion by Phil353556** [jvde-github/AIS-catcher#264] [Ideas]
Hi,
What do you think of @jvde-github?
![image](https://github.com/jvde-github/AIS-catcher/assets/64729485/6190feb9-eaca-4a59-8611-5fd6b7deb76c)

thank you
Philippe

**Comment by jvde-github** [comment]
Good idea to clean that up. I have added something along the lines of your suggestion. Hopefully that will help a few.

**Reply by Phil353556** [reply]
I hope too, and will avoid you to have trivial question :-)
